### PR TITLE
#451 Find commands wrap

### PR DESCRIPTION
**Pull request by kleaver**
https://github.com/limetext/lime/issues/451

Just giving this a shot, not sure if it's the best implementation.

### CODE REVIEW

**Comment by quarnster** [comment]
LGTM, but could you write a test case also?

**Comment by kleaver** [comment]
Tests added, but they are dependent on the location of the situation in the array full of tests, which is kind of weird.  Not sure how to easily get around that, suggestions welcome.

File: backend/commands/findreplace_test.go [hunk 1]
```diff
@@ -108,6 +125,9 @@ func runReplaceTest(tests []replaceTest, t *testing.T, commands ...string) {
 		for _, r := range test.cursors {
 			v.Sel().Add(r)
 		}
+
+		v.Settings().Set("find_wrap", tests[i].fw)
```
Comment (erbridge) [inline backend/commands/findreplace_test.go]: This can just be `test.fw`.

File: backend/commands/findreplace_test.go [hunk 2]
```diff
@@ -36,6 +37,9 @@ func runFindTest(tests []findTest, t *testing.T, commands ...string) {
 		for _, r := range test.in {
 			v.Sel().Add(r)
 		}
+
+		v.Settings().Set("find_wrap", tests[i].fw)
```
Comment (erbridge) [inline backend/commands/findreplace_test.go]: Same here.

**Comment by erbridge** [comment]
LGTM, thanks.

**Pull request merged**